packages/ember-htmlbars/tests/integration/mount_test.js

```diff
@@ -0,0 +1,129 @@
+import Application from 'ember-application/system/application';
+import Engine from 'ember-application/system/engine';
+import EmberObject from 'ember-runtime/system/object';
+import { compile } from 'ember-htmlbars-template-compiler';
+import ComponentLookup from 'ember-views/component_lookup';
+import Component from 'ember-htmlbars/component';
+import Controller from 'ember-runtime/controllers/controller';
+import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
+import run from 'ember-metal/run_loop';
+import { OWNER, getOwner } from 'container/owner';
+import isEnabled from 'ember-metal/features';
+import { getEngineParent } from 'ember-application/system/engine-parent';
+
+let App,
+    app,
+    appInstance,
+    ChatEngine,
+    chatEngineResolutions;
+
+function commonSetup() {
+  App = Application.extend({ router: null });
+
+  let ChatEngineResolver = EmberObject.extend({
+    resolve(fullName) {
+      return chatEngineResolutions[fullName];
+    }
+  });
+
+  ChatEngine = Engine.extend({
+    router: null,
+    Resolver: ChatEngineResolver
+  });
+
+  run(function() {
+    app = App.create();
+    app.register('engine:chat', ChatEngine);
+    app.registerOptionsForType('template', { instantiate: false });
+    app.registerOptionsForType('component', { singleton: false });
+    app.register('component-lookup:main', ComponentLookup);
+
+    appInstance = app.buildInstance();
+  });
+}
+
+function commonTeardown() {
+  runDestroy(appInstance);
+  runDestroy(app);
+  app = appInstance = null;
+}
+
+if (isEnabled('ember-application-engines')) {
+  QUnit.module('mount keyword', {
+    setup() {
+      commonSetup();
+    },
+
+    teardown() {
+      commonTeardown();
+    }
+  });
+
+  QUnit.test('asserts that only a single param is passed', function(assert) {
```

Comment:
I think you will need to change these to use `import test from 'skip-if-glimmer';` for it to pass on Travis.